frontend-api/src/test/java/uk/gov/di/authentication/frontendapi/lambda/ResetPasswordHandlerTest.java

```diff
@@ -392,7 +391,6 @@ void shouldReturn400WhenUserHasInvalidSession() {
     @Test
     void shouldNotUpdateAccountModifiersTableWhenPasswordResetSuccessfullyButFeatureIsDisabled()
```

Comment:
Should this test just be deleted?

Reply:
yeah was originally going to rename it but seen the subsequent test covers the unverified phone scenario